CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ __New Features__
 
 * New API endpoint for retrieving a nested page tree (#1155)
   `api/pages/nested` returns a nested JSON tree of all pages.
+  Allot template configs to declare which user types can make edits/delete.
```

Comment:
Typo: Allow

Also I think

"Allow to declare which user role can edit page content on the page layout level"

explains better what the change is about.